sfge/src/test/java/com/salesforce/graph/symbols/apex/IndeterminantTest.java

```diff
@@ -37,7 +37,7 @@ public void setup() {
      */
     @ValueSource(
             strings = {"Boolean", "Id", "Integer", "List<String>", "Map<String, String>", "String"})
-    @ParameterizedTest(name = "{0}")
+    @ParameterizedTest(name = "{displayName}: variableType = {0}")
```

Comment:
An early attempt at the fix broke some of the tests in this file, but it was hard to tell which ones without `{displayName}` in the name, so I added that.